lib/cusolver/dense.jl

```diff
@@ -117,7 +117,7 @@ for (bname, fname,elty) in ((:cusolverDnSgetrf_bufferSize, :cusolverDnSgetrf, :F
                             (:cusolverDnCgetrf_bufferSize, :cusolverDnCgetrf, :ComplexF32),
                             (:cusolverDnZgetrf_bufferSize, :cusolverDnZgetrf, :ComplexF64))
     @eval begin
-        function getrf!(A::StridedCuMatrix{$elty})
+        function LAPACK.getrf!(A::StridedCuMatrix{$elty})
```

Comment:
I'm not sure we want to directly implement LAPACK methods. Those interfaces aren't meant to be platform independent, so isn't it better to keep our own `getrf!` and add an implementation of `LinearAlgebra.lu!`?